manual/mapper/daos/README.md

```diff
@@ -48,4 +48,68 @@ ProductDao dao = inventoryMapper.productDao("someKeyspace");
 
 The returned object is thread-safe, and can securely be shared throughout your application.
 
+### Inheritance
```

Comment:
I considered making a dedicated page for entity/dao inheritance, but wasn't sure if it'd be worth it or not, instead I added a section for each entity & dao.